src/MultiLevelPoisson.jl

```diff
@@ -83,19 +85,18 @@ end
 mult!(ml::MultiLevelPoisson,x) = mult!(ml.levels[1],x)
 residual!(ml::MultiLevelPoisson,x) = residual!(ml.levels[1],x)
 
-function solver!(ml::MultiLevelPoisson;log=false,tol=1e-3,itmx=32)
+function solver!(ml::MultiLevelPoisson;tol=2e-4,itmx=32)
     p = ml.levels[1]
     BC!(p.x;perdir=p.perdir)
-    residual!(p); r₂ = L₂(p)
-    log && (res = [r₂])
+    residual!(p); r₀ = r₂ = L∞(p); r₂₀ = L₂(p)
```

Comment:
@marinlauber  Why are there three computed residuals? It looks like only r₂ is used.